crates/test-programs/src/bin/preview1_fd_filestat_set.rs

```diff
@@ -30,14 +30,16 @@ unsafe fn test_fd_filestat_set(dir_fd: wasi::Fd) {
 
     // Check fd_filestat_set_times
     let old_atim = stat.atim;
-    let new_mtim = stat.mtim - 100;
+    let new_mtim = stat.mtim - 2000;
```

Comment:
With 100us accuracy in Node this gets rounded to no change, right? Could this -2000 be increased so the mtim is changed in node?

Reply:
2000 for u64 is 2000 * 100 nanoseconds is 200 microseconds. So I think this is the right accuracy. Reducing the order by one leads to the times being equal in the check. So I think this is right.

Reply:
Ahh I misread the documentation here that the units were 100 nanoseconds.

The actual units are therefore microsecond accuracy and this is 2 microseconds.